### PR TITLE
Add digest headers draft.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Fix: #822. 

Adds the digest-headers draft to the repository.

The original repository, with the issues to be migrated, is here:

  - https://github.com/ioggstream/draft-polli-resource-digests-http

### CODE REVIEW

File: draft-ietf-httpbis-digest-headers.md [hunk 1]
```diff
@@ -0,0 +1,893 @@
+---
+title: Resource Digests for HTTP
+abbrev: RDHTTP
```
Comment (reschke) [inline draft-ietf-httpbis-digest-headers.md]: Remove abbreviation; title is short enough.

File: draft-ietf-httpbis-digest-headers.md [hunk 2]
```diff
@@ -0,0 +1,893 @@
...
+docname: draft-polli-resource-digests-http-latest
```
Comment (reschke) [inline draft-ietf-httpbis-digest-headers.md]: need to be draft-ietf-httpbis...

File: draft-ietf-httpbis-digest-headers.md [hunk 3]
```diff
@@ -0,0 +1,893 @@
...
+category: std
+
+ipr: trust200902
+area: General
```
Comment (reschke) [inline draft-ietf-httpbis-digest-headers.md]: Applications and Real-Time

File: draft-ietf-httpbis-digest-headers.md [hunk 4]
```diff
@@ -0,0 +1,893 @@
...
+workgroup:
```
Comment (reschke) [inline draft-ietf-httpbis-digest-headers.md]: HTTP

File: draft-ietf-httpbis-digest-headers.md [hunk 5]
```diff
@@ -0,0 +1,893 @@
...
+keyword: Internet-Draft
```
Comment (reschke) [inline draft-ietf-httpbis-digest-headers.md]: remove

File: draft-ietf-httpbis-digest-headers.md [hunk 6]
```diff
@@ -0,0 +1,893 @@
...
+--- note_Note_to_Readers
+
+*RFC EDITOR: please remove this section before publication*
+
+Discussion of this draft takes place on the HTTP working group mailing list
+(ietf-http-wg@w3.org), which is archived at
+<https://lists.w3.org/Archives/Public/ietf-http-wg/>.
+
+The source code and issues list for this draft can be found at
+<https://github.com/ioggstream/draft-polli-resource-digests-http>.
```
Comment (reschke) [inline draft-ietf-httpbis-digest-headers.md]: need to adjust for HTTP WG Github.

File: draft-ietf-httpbis-digest-headers.md (hunk 5 above)
Comment (ioggstream) [reply draft-ietf-httpbis-digest-headers.md]: should I put something else here, eg. "Digest" ?

Comment (reschke) [reply draft-ietf-httpbis-digest-headers.md]: Sure.

**Comment by mnot** [comment]
Looks good to me.

**Comment by ioggstream** [comment]
@mnot Do you or @reschke  want to merge? Can I proceed by myself?

**Comment by mnot** [comment]
@ioggstream give it a try.

**Pull request merged**